frontend/components/commons/sidebar/SidebarButton.vue

```diff
@@ -60,6 +59,11 @@ export default {
 
 <style lang="scss" scoped>
 .sidebar-button {
+  display: flex;
+  outline: none;
+  border: none;
+  background: none;
+  margin: auto;
```

Comment:
is it possible to create a scss mixins to reused the code for reinitiate the outline/backgrounbd/border-margin for button ? this could be reused several time in this issue too https://github.com/argilla-io/argilla/issues/2138

Reply:
yes, sure!